templates/stamp/mysql-ubuntu-install.sh

```diff
@@ -0,0 +1,380 @@
+#!/bin/bash
```

Comment:
Initial stab from https://github.com/Azure/azure-quickstart-templates/blob/master/mysql-replication/azuremysql.sh but it's so far off it's practically done from scratch.